VL.ImGui/src/Helper/RenderHelper.cs

```diff
@@ -72,9 +72,13 @@ public static void HandleNotification(this ImGuiIOPtr _io, INotification notific
                     };
                 }
 
-                // The up & down event methods don't take the position as an argument. Therefor make sure it's present, or we end up with wrong clicks when using touch devices.
-                var pos = useWorldSpace ? mouseNotification.PositionInWorldSpace.FromHectoToImGui() : mouseNotification.Position.ToImGui();
-                _io.AddMousePosEvent(pos.X, pos.Y);
+                var flag = ImGui.GetIO().ConfigFlags;
```

Comment:
If viewports are enabled, ImGui expects the mouse position in DesktopSpace/ScreenSpace and no longer in WindowSpace

I derived this from the viewport example from ImGui.net

Reply:
Ok but the code comment says it's about the position on touch down events - not about spaces.

Reply:
That was an old code comment from you ... maybe I should have put it out there
I should clean it up again

Reply:
I know I wrote it. The comment says that the MouseMove is necessary to allow TouchDown events to be on the right spot, because the down event itself doesn't take a position. You surrounded the whole part in an if statement, which introduces different behavior. You say it was necessary because ImGui expects the positions to be in screen space, in which case I'd argue that we need to feed the right coordinates to the move event, but not skip it entirely.

Reply:
Imgui, in viewport mode, expects a mouse position in the ScreenSpace for each frame. This is set in ImGuiWindows.cs SetPerFrameImGuiData(). 

I can therefore also put it in an If at this point, as it is set anyway

Reply:
Aha, that solves the puzzle. Then either add that piece of info to the if statement or get rid of it entirely and set the mouse position in each frame in existing mode.